duckduckgo/iOS
# Update navigation to make fire more prominent

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/394139777618003

**Description**:
Updates the app navigation to make the "fire" button more prominent and for better consistency

**Steps to test this PR**:
Run the app and ensure that
- there is now a fire button on the home screen
- the home screen and the browsing screen display the same navigation options
- check that all the buttons work as expected (back, forward, fire, bookmarks, tabs, settings) as these have moved around a bit

NOTE: The fire animation is still a placeholder and will be completely rewritten prior release - we're waiting for an external company to produce the assets for us.

###### Reviewer Checklist:


###### PR DRI Checklist:
